src/gamercoder215/smpcore/utils/ItemsManager.java

```diff
@@ -0,0 +1,104 @@
+package com.stephen.plugin;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.bukkit.Material;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.attribute.AttributeModifier;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+public class ItemsManager {
+
+	public static ItemStack ThrowingKnife;
+	
+	public static ItemStack ThrowingKnife2;
+	
+	public static ItemStack ThrowingKnife3;
+	
+	public static ItemStack ThrowingKnife4;
+	
+	
+	
+	public static void init() {
+		createThrowingSword();
+		createThrowingSword2();
+		createThrowingSword3();
+		createThrowingSword4();
+	}
+	
+	private static void createThrowingSword() {
+		ItemStack item = new ItemStack(Material.STICK, 64);
```

Comment:
I would change this to the item that is hooked to the armor stand to be more user-friendly
Same for the rest..